js/src/collect/collect-v5.js

```diff
@@ -10,12 +11,14 @@ module.exports = function() {
 	var outerFields = [];
 	var acfFields = [];
 
-	if ( wp.data.select( "core/block-editor" ) ) {
-		// Return only fields in metabox areas (either below or side) or
-		// ACF block fields in the content (not in the sidebar, to prevent duplicates)
+	// Check whether classic editor is used.
+	if ( window.wpseoScriptData.isBlockEditor ) {
 		var parentContainer = jQuery( ".metabox-location-normal, .metabox-location-side, .acf-block-component.acf-block-body" );
 		acfFields = acf.get_fields( false, parentContainer );
```

Comment:
```suggestion
        // Check if Gutenberg editor is used.
	if ( window.wpseoScriptData.isBlockEditor ) {
	        /*
	         * Return only fields in metabox areas (either below or side) or
	         * ACF block fields in the content (not in the sidebar, to prevent duplicates).
		*/
		var parentContainer = jQuery( ".metabox-location-normal, .metabox-location-side, .acf-block-component.acf-block-body" );
		acfFields = acf.get_fields( false, parentContainer );
```